packages/apollo-gateway/src/QueryPlan.ts

```diff
@@ -40,16 +39,9 @@ export interface FetchNode {
   kind: 'Fetch';
   serviceName: string;
   selectionSet: SelectionSetNode;
-  variableUsages?: VariableUsage[];
+  variableUsages?: { [name: string]: VariableDefinitionNode };
```

Comment:
Alternatively, this could also be `VariableDefinitionNode[]` because the name is part of the node.